test/core/gen2/test_config.py

```diff
@@ -56,6 +56,7 @@ def test_from_dict(self):
                              spatial_res=0.05,
                              time_range=['2018-01-01', None],
                              time_period='4D',
+                             temporal_resampling='slinear',
```

Comment:
See comment above, make this a config object.

Reply:
Closing to keep the conversation in one comment.